### DISCUSSION TITLE
Whats the best way to show/hide a table column based on the tab selected?

### DISCUSSION THREAD

**Discussion by rappasoft** [filamentphp/filament#14787] [Help] [answered]
I feel like i've had this issue a bunch and never figured it out fully.

Basically you have a resource, and in your ListRecords class you define some tabs right.

But what if on your table, you want to conditionally show columns based on the selected tab?

You can use `request()->get('activeTab')` and check, however it doesn't persist across tabs if you switch back and forth, it only works on the initial page load.

Any ideas?

**Comment by leandrocfe** [comment, marked as answer]
Inject `$livewire` and use `$livewire->activeTab`

**Reply by rappasoft** [reply]
You are correct i'm annoyed I didn't think of that. Thanks!
